src/site/content/en/patterns/image-demo-example/demo.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: demo
+patternId: image-demo-example
+---
+
+<figure >
```

Comment:
I figured you might eventually add this (and it works with `{{ content | safe }}`) :)

Reply:
Yeah :)

Reply:
This doesn't seem to render, so I'm confused:
https://deploy-preview-5994--web-dev-staging.netlify.app/patterns/image-demo-example/demo

(Whereas sample.html is included in the output; although I appreciate that might be sort of unintentional as it is inlined.)

Reply:
Ah good catch, it is because an unclean merge from main in the meantime. Let me fix it.